docker/README.md

```diff
@@ -205,5 +205,29 @@ curl "http://localhost:9090/npg_ranger/sample?accession=NA30000M&format=sam" -o
 curl "http://localhost:9090/npg_ranger/ga4gh/sample/NA30000M?format=sam" -o NA30000M.sam # This returns a htsget JSON
 ```
 
+## If the .cram md5 file test failed
+Note that the md5sum test for the temp.cram file above may fail.
+If it does, while the .sam and .bam file tests do not, it may be due to an
+old version of Samtools being used.
+This issue was spotted while testing the validity of the same file being
+downloaded and compared from two different sources, using different
+a new and an old version of Samtools, resulting in differences in the data
+beyond the headers being different.
+One way to get around this and to make sure that they are the same is to use 
+bamseqchksum, but this requires a REF_PATH to cram_cache to be set, done
+locally on the farm.
+
+To do this:
+```
+# Make sure the files are downloaded locally.
+curl "http://localhost:9090/npg_ranger/sample?accession=NA12878&format=cram&region=22:16100000-16105000" -o testingCram.cram
+
+# Allows for bamseqchksum to work on this example.
+export REF_PATH=[path to cram_cache]
+
+# Check that this is the same as e9411b620538bcb92a0a7749682ea326
+cat testingCram.cram | bamseqchksum inputformat=cram | md5sum
+```
+
 These dockerfiles are a proof of concept only; no security has been enabled on
-them, and are obviously not production-ready.
+them, and are obviously not production-ready.
```

Comment:
Just curious what changed in this line. I cannot see differences.

Reply:
As far as I can tell from comparing this branch to another, I think I removed a newline at the end of the file